controller/transfer/record_transfer_manager.py

```diff
@@ -90,25 +69,80 @@ def import_file(project_id: str, task: UploadTask) -> None:
             create_attributes_and_get_text_attributes(project_id, records_data)
             primary_keys = attribute.get_primary_keys(project_id)
 
-        import_tasks_and_labels_pipeline(project_id=project_id, tasks_data=tasks_data)
+        import_labeling_tasks_and_labels_pipeline(
+            project_id=project_id, tasks_data=tasks_data
+        )
         import_records_and_rlas_pipeline(
-            user_id=task.user_id,
+            user_id=user_id,
             project_id=project_id,
             records_data=records_data,
             labels_data=labels_data,
             category=record_category,
             primary_keys=primary_keys,
         )
 
-        progress = ((idx + 1) / chunks_count) * 100
-        task_manager.update_task(project_id, task.id, progress=progress)
+        if upload_task is not None:
+            progress = ((idx + 1) / chunks_count) * 100
+            upload_task_manager.update_task(
+                project_id, upload_task.id, progress=progress
+            )
+
+
+def import_record_json(
+    project_id: str, user_id: str, record_data: Dict[str, Any]
+) -> None:
+    import_records_and_rlas(project_id, user_id, record_data)
+    token_manager.request_tokenize_project(project_id, user_id)
+    token_manager.create_rats_entries(project_id, user_id)
```

Comment:
shouldn't be nessecary since the tokenizer does it after the doc creation anyway and cancels the job if doc creation is running

Reply:
When I did not add it, I had a loading spinner in the UI for this record and received the error in the gateway that there was not docbin entry for said record.